packages/docusaurus-theme-classic/src/theme/BlogPostItem/Footer/index.tsx

```diff
@@ -42,8 +50,18 @@ export default function BlogPostItemFooter(): JSX.Element | null {
       )}
 
       {isBlogPostPage && editUrl && (
-        <div className="col margin-top--sm">
+        <div
+          className={clsx(
+            'col margin-top--sm',
+            styles.blogPostFooterLastUpdated,
+          )}>
           <EditThisPage editUrl={editUrl} />
+          {(lastUpdatedAt || lastUpdatedBy) && (
+            <LastUpdated
+              lastUpdatedAt={lastUpdatedAt}
+              lastUpdatedBy={lastUpdatedBy}
+            />
+          )}
```

Comment:
This will not display the last update unless there's an edit url.

We want the same design as docs, so why not using the exact same code (using row/col instead of a custom style)

Maybe we could extract a shared component that works for both docs and blog?

```tsx
export function EditMetaRow({
  className,
  editUrl,
  lastUpdatedAt,
  lastUpdatedBy,
}: EditMetaRowProps) {
  return (
    <div className={clsx('row',className)}>
      <div className="col">{editUrl && <EditThisPage editUrl={editUrl} />}</div>
      <div className={clsx('col', styles.lastUpdated)}>
        {(lastUpdatedAt || lastUpdatedBy) && (
          <LastUpdated
            lastUpdatedAt={lastUpdatedAt}
            lastUpdatedBy={lastUpdatedBy}
          />
        )}
      </div>
    </div>
  );
}
```

```tsx
const canDisplayEditMetaRow = !!(editUrl || lastUpdatedAt || lastUpdatedBy);
{canDisplayEditMetaRow && (
  <EditMetaRow
    editUrl={editUrl}
    lastUpdatedAt={lastUpdatedAt}
    lastUpdatedBy={lastUpdatedBy}
  />
)}
```

I don't know if `EditMetaRow` is a good name but this is what we have for docs atm.